lib/config-validator.ts

```diff
@@ -17,17 +20,24 @@ async function validate(
   config: RenovateConfig,
   isPreset = false
 ): Promise<void> {
-  const res = await validateConfig(massageConfig(config), isPreset);
-  if (res.errors.length) {
-    console.log(
-      `${desc} contains errors:\n\n${JSON.stringify(res.errors, null, 2)}`
+  const { isMigrated, migratedConfig } = migrateConfig(config);
+  if (isMigrated) {
+    logger.info(
```

Comment:
I think we should warn here to have more visibillity
```suggestion
    logger.warn(
```